data/recipes/upload_web_ts.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "upload_web_ts",
+    "description": "Uploads a parsed web history or EVTX records stored in a CSV or Plaso file to Timesketch.",
+    "short_description": "Uploads a CSV/JSONL or Plaso file to Timesketch.",
+    "modules": [{
+        "wants": [],
+        "name": "FilesystemCollector",
+        "args": {
+            "paths": "@files"
+        }
+    }, {
+        "wants": ["FilesystemCollector"],
+        "name": "TimesketchExporter",
+        "args": {
+            "incident_id": "@incident_id",
+            "token_password": "@token_password",
+            "sketch_id": "@sketch_id",
+	    "analyzers": "browser_search,browser_timeframe,account_finder,phishy_domains,evtx_gap,login,win_crash,safebrowsing,chain,"
+        }
+    }, {
+        "wants": ["TimesketchExporter"],
+        "name": "TimesketchEnhancer",
+        "args": {
+	    "wait_for_analyzers": "@wait_for_analyzers",
+            "searches_to_skip": "@searches_to_skip",
+            "aggregations_to_skip": "@aggregations_to_skip",
+            "include_stories": "@timesketch_include_stories",
+            "token_password": "@token_password",
+	    "max_checks": "@analyzer_max_checks",
+	    "formatter": "html"
+	}
+    }],
+    "args": [
+        ["files", "Comma-separated list of paths to CSV files or Plaso storage files", null],
+        ["--sketch_id", "Sketch to which the timeline should be added", null],
+        ["--token_password", "Optional custom password to decrypt Timesketch credential file with", ""],
+        ["--incident_id", "Incident ID (used for Timesketch description)", null],
+	["--wait_for_analyzers", "Wait for analyzers until they complete their run, if set to False the TS enhancer will be skipped", true],
+	["--timesketch_include_stories", "Include story dumps in reports", false],
+	["--searches_to_skip", "A comma separated list of saved searches that should not be uploaded.", null],
+	["--analyzer_max_checks", "Number of wait cycles (per cycle is 3 seconds) before terminating wait for analyzers to complete.", 0],
+	["--aggregations_to_skip", "A comma separated list of aggregation names that should not be uploaded.", null]
+    ]
```

Were you mixing tabs and spaces? 🤔


```suggestion
    "args": [
        ["files", "Comma-separated list of paths to CSV files or Plaso storage files", null],
        ["--sketch_id", "Sketch to which the timeline should be added", null],
        ["--token_password", "Optional custom password to decrypt Timesketch credential file with", ""],
        ["--incident_id", "Incident ID (used for Timesketch description)", null],
        ["--wait_for_analyzers", "Wait for analyzers until they complete their run, if set to False the TS enhancer will be skipped", true],
        ["--timesketch_include_stories", "Include story dumps in reports", false],
        ["--searches_to_skip", "A comma separated list of saved searches that should not be uploaded.", null],
        ["--analyzer_max_checks", "Number of wait cycles (per cycle is 3 seconds) before terminating wait for analyzers to complete.", 0],
        ["--aggregations_to_skip", "A comma separated list of aggregation names that should not be uploaded.", null]
    ]
```